app/views/notifications/_feed.html.erb

```diff
@@ -1,21 +1,3 @@
 <% notifications.each do |notification| %>
```

Comment:
do we need the `_feed` partial? You can pass a notifications Array to `render` and rails will find the right partial.